src/Middleware/ImplicitHeadMiddleware.php

```diff
@@ -18,6 +18,7 @@
 use Psr\Http\Message\StreamInterface;
 use Psr\Http\Server\MiddlewareInterface;
 use Psr\Http\Server\RequestHandlerInterface;
+use zend\expressive\router\routeresult as ZendExpressiveRouteResult;
```

Comment:
Please make sure that the case matches the real namespace.
```suggestion
use Zend\Expressive\Router\RouteResult as ZendExpressiveRouteResult;
```